ext/php5/priority_sampling/priority_sampling.c

```diff
@@ -28,6 +90,7 @@ static bool dd_rule_matches(zval *pattern, zval *prop TSRMLS_DC) {
 
 static void dd_decide_on_sampling(ddtrace_span_fci *span TSRMLS_DC) {
     int priority = DDTRACE_G(default_priority_sampling);
+    enum dd_sampling_mechanism mechanism = DD_MECHANISM_MANUAL;
```

Comment:
Because it is. This function is the automatic mechanism. The only reason it should be not unknown, is if it was changed by the user before the automatic mechanism kicks in.